sonar-dotnet-tests-library/src/main/java/org/sonar/plugins/dotnet/tests/DotCoverReportsAggregator.java

```diff
@@ -66,7 +65,7 @@ public void accept(File file, Coverage coverage) {
     try {
       pathStream = Files.list(folder.toPath());
     } catch (IOException e) {
-      throw Throwables.propagate(e);
+      throw new RuntimeException(e);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00112)